### ISSUE TITLE
Transactions are not propagated when running a private chain

### ISSUE THREAD

**Issue by nnicksize** [status-im/status-go#169]
**geth version**
```
Geth
Version: 1.6.2-unstable
Git Commit: 10cd54e92464ed8bb77b2e8e8dfb62fe91353ce9
Architecture: amd64
Protocol Versions: [63 62]
Network Id: 1
Go Version: go1.8.1
Operating System: windows
```

**status-go version**
2f0c93f (refactor: linters + recommendations applied, closes #146 )

**Problem**
When connecting a status-go node to a private test chain (running multiple geth nodes), transactions from that status-go node are not propagated to any peers.

**Additional info**
- There are multiple mining nodes, so it's probably not connected to [this](https://github.com/ethereum/go-ethereum/issues/2769) or similar issues.
- The status-go node is connected to multiple peers.
- Sending transactions from ```geth --light``` nodes on the network works fine.
- running eth.pendingTransactions on the status-go node shows the transactions.
- connection to the private chain has been done bye changing the generated genesis block in status-go/geth/params/config.go.

**Comment by farazdagi** [comment]
Thanks for your report. Till this point we haven't tried using Status on private net, although since we are at the moment preparing Network Switching feature (where we allow users to switch between Ropsten/Rinkeby/Homestead), I think it is a good time to review how everything works with private test nets.

**Comment by nnicksize** [comment]
This seems to be resolved in the latest commit [b3a56eb](https://github.com/status-im/status-go/commit/b3a56eb00b97d853edcbcb4fb2cf8803fe7a0370).
